webproject/src/main/webapp/js/myRooms.js

```diff
@@ -7,9 +7,10 @@ async function getListings() {
     for (var i = 0; i < entries.length; i++) {
         let room = entries[i];
         var roomData = room[1];
-        let isRoomJoinedResponse = await fetch("/join?roomId=" + room[0]);
-
-        if (roomData.isOpen && isRoomJoinedResponse.text() !== "Join") {
+        let isRoomJoinedResponse = await fetch("/userRoomStatus?roomId=" + room[0]);
+        let res = await isRoomJoinedResponse.text();
+        console.log(res);
```

Comment:
Remove console.log

Reply:
Well spotted, thank you! I hadn't noticed.